runbot_travis2docker/models/runbot_build.py

```diff
@@ -69,6 +69,7 @@ def _get_introspection(self):
         help='Dockerfile path created by travis2docker')
     docker_image = fields.Char(help='New image name to create')
     docker_container = fields.Char(help='New container name to create')
+    sync_weblate = fields.Boolean('Synchronized with weblate', readonly=True)
```

Comment:
Please, don't use `readonly=True` to test faster this case creating a new build...
Just use `copy=False`